src/pages/index.js

```diff
@@ -1,118 +1,173 @@
+// Importamos estilos e imágenes
 import "../pages/index.css";
-
-import {initialCards,
-  cardListEl,
-  editProfileModal,
-  editProfileForm,
-  addCardForm,
-  addCardModal,
-  addCardButton,
-  profileEditButton,
-  profileTitle,
-  profileDescription,
-  nameInput,
-  descriptionInput,
-  formValidationSettings,
+import logo from '../images/logo.svg';
+import avatar from '../images/jacques-cousteau.png';
+import editIcon from '../images/Edit_pencil.svg';
+
+document.querySelector('.header__logo').src = logo;
+document.querySelector('.profile__image').src = avatar;
+document.querySelector('.profile__edit-image-icon').src = editIcon;
+
+import { 
+    editProfileForm,
+    addCardForm,
+    addCardButton,
+    profileEditButton,
+    profileTitle,
+    profileDescription,
+    nameInput,
+    descriptionInput,
+    formValidationSettings
 } from "../utils/constants.js";
 
 import FormValidator from "../components/FormValidator.js";
-
 import Card from "../components/Card.js";
-
-import PopupWithImage from "../components/PopupWithImage.js";
-
 import PopupWithForm from "../components/PopupWithForm.js";
-
+import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
-
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
+
+const api = new Api({
+    baseUrl: 'https://around-api.en.tripleten-services.com/v1',
+    headers: {
+        authorization: "77f6fa17-673a-40ba-89df-6b222f24f8b5",
+        'Content-Type': 'application/json'
+    }
+});
 
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
-);
+const userInfo = new UserInfo(profileTitle, profileDescription, document.querySelector('.profile__image'));
 
-section.renderItems();
+// Inicializar perfil
+api.getUserInfo()
+    .then(userData => {
+        userInfo.setUserInfo(userData);
+        userInfo.setAvatar(userData.avatar); 
+    })
+    .catch(err => console.error('Error loading user info:', err));
 
+// Crear cards usando el método renderItems de Section
+const section = new Section(
+    {
+        items: [],
+        renderer: (cardData) => {
+            const cardElement = createCard(cardData);
+            if (cardElement) {
+                section.addItem(cardElement);
+            }
+        }
+    },
+    ".cards__list"
+);
 
-function createCard(data) {
-  const cardElement = getCardElement(data);
-  section.addItem(cardElement);
+api.getInitialCards()
+    .then(cardsData => {
+        section._items = cardsData; // Asignar datos obtenidos a _items
```

Comment:
`_items`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.